common/tools/dev-tool/src/util/prettier.ts

```diff
@@ -2,6 +2,12 @@ import * as prettier from "prettier";
 
 import prettierOptions from "../../../eslint-plugin-azure-sdk/prettier.json";
 
-export function format(contents: string, parser?: prettier.BuiltInParserName): string {
-  return prettier.format(contents, { ...(prettierOptions as prettier.Options), parser: parser });
+export async function format(
+  contents: string,
+  parser?: prettier.BuiltInParserName,
+): Promise<string> {
+  return await prettier.format(contents, {
```

Comment:
again no need to `return await` can just return

```suggestion
  return prettier.format(contents, {
```